jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/ProgressOutputStream.kt

```diff
@@ -32,7 +32,7 @@ class ProgressOutputStream(private val outputStream: OutputStream, val size: Lon
     }
 
     override fun write(b: ByteArray?, off: Int, len: Int) {
-        outputStream.write(b, off, len)
+        b?.let { outputStream.write(b, off, len) }
```

Comment:
Should we allow null?

Reply:
This is actually annotated `@notnull`, so changed to not allow null